FutureNorthants/northants-website
# Upgrade drupal to 9.3.7

Update [drupal](https://www.drupal.org/project/drupal/releases/9.3.7) and also update `drupal/paragrahs` and `drupal/entity` to latest versions.

## To Test
- Checkout this branch and run `make` to install the composer dependencies
- `./drush.wrapper @docker cim` should have no changes
- `./drush.wrapper @docker updb` should also have no changes
- Test out as much as possible in Drupal
- Launch the frontend with preprod_frontend branch and test that everything works as expected without any changes
